src/pds/registrysweepers/ancestry/__init__.py

```diff
@@ -73,6 +83,15 @@ def run(
             updates,
             index_name="registry",
         )
+        log.info("Generating updates from deferred records...")
```

Comment:
Instead of three characters `...` you can use a single ellipsis character `…`, a savings of 2 characters! 😝